### ISSUE TITLE
allowing folder hard links to sync

### ISSUE THREAD

**Issue by finius** [owncloud/client#6189]
I would suggest to allow syncing of hard links to the ownCloud client. At least as an opt-in feature, so that a copy of a users client folder can be synced without having to rearrange local folder usage nor reconfigure the sync configuration itself.

**Comment by ogoffart** [comment]
We do sync hard links since version 2.0 of the client.  (https://github.com/owncloud/client/pull/3388)

Please be specific in what is not working, with steps to reproduce.

**Comment by finius** [comment]
Thanks for your feedback. I wasn't aware that this should be possible, as I tried it today and the client tells in the log that the hard or soft linked folder can't be synced as "symbolic links are not supported for synchronisation".
I use Windows 10 with latest build of Nextcloud Windows client.
I set up a fresh sync with a new user, removed the default skeleton files and made some new links to local folders within the NextCloud sync directory, like:
**mklink /J "C:\Users\john.doeNextcloud\Desktop" "C:\Users\john.doe\Desktop"** 
The clients sees the link but logs that links are not supported for sync and ignores it. 
<img width="1038" alt="bildschirmfoto 2017-11-24 um 10 53 10" src="https://user-images.githubusercontent.com/5287181/33205011-b39baffa-d105-11e7-9f76-91910b806f93.png">
